test/e2e/page-objects/pages/homepage.ts

```diff
@@ -0,0 +1,168 @@
+    this.driver = driver;
+    this.headerNavbar = new HeaderNavbar(driver);
+    this.sendButton = '[data-testid="eth-overview-send"]';
+    this.activityTab = '[data-testid="account-overview__activity-tab"]';
+    this.tokensTab = '[data-testid="account-overview__asset-tab"]';
+    this.confirmedTransactions = {
+      text: 'Confirmed',
+      css: '.transaction-status-label--confirmed',
+    };
+    this.balance = '[data-testid="eth-overview__primary-currency"]';
+    this.completedTransactions = '[data-testid="activity-list-item"]';
+    this.transactionAmountsInActivity =
+      '[data-testid="transaction-list-item-primary-currency"]';
+  }
+
+  async check_pageIsLoaded(): Promise<void> {
+    try {
+      await this.driver.waitForSelector(this.sendButton);
+      await this.driver.waitForSelector(this.activityTab);
+      await this.driver.waitForSelector(this.tokensTab);
+    } catch (e) {
+      console.log('Timeout while waiting for home page to be loaded', e);
+      throw e;
+    }
+    console.log('Home page is loaded');
+  }
+
+  async check_expectedBalanceIsDisplayed(
+    expectedBalance: string = DEFAULT_GANACHE_ETH_BALANCE_DEC,
+  ): Promise<void> {
+    try {
+      await this.driver.waitForSelector({
+        css: this.balance,
+        text: `${expectedBalance} ETH`,
+      });
+    } catch (e) {
+      const actualBalance = await this.getBalance();
+      const errorMessage = `Expected balance ${expectedBalance} ETH, got balance ${actualBalance} ETH`;
+      console.log(errorMessage, e);
+      throw e;
+    }
+    console.log(
+      `Expected balance ${expectedBalance} ETH is displayed on homepage`,
+    );
+  }
+
+  async getBalance(): Promise<number> {
+    const element = await this.driver.findElement(this.balance);
+    await this.driver.waitForNonEmptyElement(element);
+    return parseFloat(await element.getText());
+  }
+
+  async startSendFlow(): Promise<void> {
+    await this.driver.clickElement(this.sendButton);
+  }
+
+  async goToActivityList(): Promise<void> {
+    console.log(`Open activity tab on homepage`);
+    await this.driver.clickElement(this.activityTab);
+  }
+
+  /**
+   * This function checks if the specified number of confirmed transactions are displayed in the activity list on homepage.
+   * It waits up to 10 seconds for the expected number of confirmed transactions to be visible.
+   *
+   * @param expectedNumber - The number of confirmed transactions expected to be displayed in activity list. Defaults to 1.
+   * @returns A promise that resolves if the expected number of confirmed transactions is displayed within the timeout period.
+   */
+  async check_confirmedTxNumberDisplayedInActivity(
+    expectedNumber: number = 1,
+  ): Promise<void> {
+    console.log(
+      `Wait for ${expectedNumber} confirmed transactions to be displayed in activity list`,
+    );
+    await this.driver.wait(async () => {
+      const confirmedTxes = await this.driver.findElements(
```

Comment:
The short form of `Transactions` is normally `Txs`, not `Txes`

Reply:
Changed, thanks !

Reply:
It looks like there were 4 `Txes`, but you only fixed 2

Reply:
oops, yes i missed it, changed ! Thanks Howard!